src/Illuminate/Support/Str.php

```diff
@@ -222,19 +222,52 @@ public static function plural($value, $count = 2)
      */
     public static function random($length = 16)
     {
-        if (function_exists('random_bytes')) {
-            $bytes = random_bytes($length * 2);
-        } elseif (function_exists('openssl_random_pseudo_bytes')) {
-            $bytes = openssl_random_pseudo_bytes($length * 2);
-        } else {
-            throw new RuntimeException('OpenSSL extension is required for PHP 5 users.');
+        $pool = '0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ';
+
+        if (!function_exists('openssl_random_pseudo_bytes')) {
+            throw new RuntimeException('OpenSSL extension is required.');
         }
 
-        if ($bytes === false) {
-            throw new RuntimeException('Unable to generate random string.');
+        // can be replaced by random_int in  PHP 7: https://wiki.php.net/rfc/easy_userland_csprng
```

Comment:
Actually, better, please create a randomInt function on this class that first uses the php7 version, or falls back here.